packages/block-library/src/heading/deprecated.js

```diff
@@ -42,6 +42,37 @@ const blockAttributes = {
 };
 
 const deprecated = [
+	{
+		attributes: blockAttributes,
```

Comment:
I think we're missing blockSupports here

Reply:
We are not changing the block supports in this PR. If we don't pass the supports on deprecated the block should use the current ones, right? So, in this case, it may not be needed?

Reply:
It doesn't matter if we're changing it or not, we should always add it to ensure the produced markup is the same, otherwise it will fall back to default values.

Reply:
Ah, I thougths it used the current support value, not the default ones. Thank you for noticing this. The PR was updated.